Makefile

```diff
@@ -35,8 +35,10 @@ TESTFLAGS    :=
 STRESSFLAGS  :=
 DUPLFLAGS    := -t 100
 BUILDMODE    := install
+GOPATH       := $(realpath ../../../..)
```

Comment:
I agree for vendoring the important thing is that it builds correctly through go. But people might still find it useful to e.g. run `make test` to make sure what they are using is not broken.

But making the Makefile detect this is not very easy. I think what is reasonable is to do some effort to check that ../../../.. is indeed the root of a go path; if it is not just don't set GOPATH and assume it is already set correctly. I'll try that